components/ui-api-layer/docs/README.md

```diff
@@ -6,4 +6,6 @@ This directory contains the following documents that relate to the project:
 - [Configuration](./configuration.md) lists environmental variables and other parameters to configure the application
 - [Terminology](./terminology.md) defines general terms used in the project's documentation
 - [Project Structure](./project-structure.md) describes the structure of this repository
-- [Architecture](./architecture.md) presents the overall architecture of this application
\ No newline at end of file
+- [Architecture](./architecture.md) presents the overall architecture of this application
+- Queries:
```

Comment:
Why are we adding here such information? we do not have any doc for other queries. We should align that with others because if you want to add it then we should decide that this required and should be added when a new query is implemented and also create a follow-up task for already implemented queries which are not yet documented.

But for my perspective, this documentation is not necessary at all.

Reply:
it was discussed we can place it here. I do not see any other place to document the owner reference filtering logic.